frontend/src/metabase/entities/dashboards/forms.js

```diff
@@ -0,0 +1,48 @@
+import { t } from "ttag";
+import MetabaseSettings from "metabase/lib/settings";
+import { PLUGIN_CACHING } from "metabase/plugins";
+
+const FORM_FIELDS = [
+  {
+    name: "name",
+    title: t`Name`,
+    placeholder: t`What is the name of your dashboard?`,
+    autoFocus: true,
+    validate: name => (!name ? t`Name is required` : null),
+  },
+  {
+    name: "description",
+    title: t`Description`,
+    type: "text",
+    placeholder: t`It's optional but oh, so helpful`,
+  },
+  {
+    name: "collection_id",
+    title: t`Which collection should this go in?`,
+    type: "collection",
+    validate: collectionId =>
+      collectionId === undefined ? `Collection is required` : null,
```

Comment:
I might've missed this in the first round, but one more translation needed here

Reply:
Sharp eye! Thank you